Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -312,6 +312,8 @@ def parse_args(args):
         optparse.make_option("--world-leaks", action="store_true", default=False, help="Check for world leaks (currently, only documents). Differs from --leaks in that this uses internal instrumentation, rather than external tools."),
         optparse.make_option("--accessibility-isolated-tree", action="store_true", default=False, help="Runs tests in accessibility isolated tree mode."),
         optparse.make_option("--allowed-host", type="string", action="append", default=[], help="If specified, tests are allowed to make requests to the specified hostname."),
+        optparse.make_option("--signposts", action="store_true", default=False,
+            help="Gather data from performance signposts in the test-runner."),
```

Comment:
Instead of enabling/disabling signpost collection altogether, could this only control whether they are logged? Otherwise it gets difficult to follow when signposting is enabled.